lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,23 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ *
+ * @public
+ * @alias module:@ui5/builder.processors.moduleBundler~ModuleSection
+ * @typedef {object} ModuleSection
+ * @property {boolean} resolve=false Whether (transitive) dependencies of module should be resolved
+ * 		and added to the module set
+ * @property {boolean} resolveConditional Whether conditional dependencies of modules should be resolved
+ * 		and added to the module set for this section
+ * @property {boolean} renderer=false Whether renderers for controls should be added to the module set
+ * @property {boolean} sort=true Whether the modules should be sorted by their dependencies
+ * @property {string} mode The embedding technology. Either 'provided', 'raw', 'preload' or 'require'
+ * @property {string[]} filters List of modules that should be in- or exluded
```

Comment:
should be mentioned what includes and excludes are (kind of global pattern) and how includes are differentiated from excludes (leading - or exclamation mark)

Reply:
done